ajax/libs/swiftclick/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "swiftclick",
+  "description": "Eliminates the 300ms click event delay on touch devices that support orientation change",
+  "author": {
+    "name": "Ivan Hayes",
+    "url": "http://www.ivanhayes.com"
+  },
+  "homepage": "https://github.com/munkychop/swiftclick",
```

Comment:
To correct the description about the homepage, we'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.

@pvnr0082t you can write this part in the test.